service/frontend/admin/adminGrpcHandler.go

```diff
@@ -29,174 +29,174 @@ import (
 	"github.com/uber/cadence/common/types/mapper/proto"
 )
 
-type adminGRPCHandler struct {
-	h AdminHandler
+type AdminGRPCHandler struct {
+	h Handler
 }
 
-func newAdminGRPCHandler(h AdminHandler) adminGRPCHandler {
-	return adminGRPCHandler{h}
+func NewAdminGRPCHandler(h Handler) AdminGRPCHandler {
+	return AdminGRPCHandler{h}
 }
 
-func (g adminGRPCHandler) register(dispatcher *yarpc.Dispatcher) {
+func (g AdminGRPCHandler) Register(dispatcher *yarpc.Dispatcher) {
```

Comment:
I am guessing this needs to be publically exposed because it's moved into a subpackage?

Reply:
Yes.